task/kraken-scenario/0.1/kraken-scenario.yaml

```diff
@@ -0,0 +1,57 @@
+
+---
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: kraken-scenario
+  labels:
+    app.kubernetes.io/version: "0.1"
+    app.kubernetes.io/name: "kraken-scenario-task"
+    tool: kraken
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/displayName: kraken-scenario
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/platforms: linux/amd64
+    tekton.dev/tags: kraken, chaos, test
+spec:
+  description: >-
+    This `task` can be used to start a kraken chaos scenarios as part of a chaos pipeline.
+
+    kraken chaos testing is a good way to execute chaos testing of openshift or kubernetes
+    cluster as well as application running on top of it. This task allow you to use kraken 
+    chaos scenarios as part of your application testing pipeline. It can also be part of
+    a complete chaos pipeline for your infrastructure layer.
+  params:
+    - name: awsCredentialSecret
+      type: string
+      description: Name of the secret key holding the aws-credentials (mandatory but used only for aws infrastructure chaos test).
+      default: kraken-aws-creds
+  steps:
+    - image: quay.io/chaos-kubox/krkn:latest
+      args:
+        - python3 run_kraken.py -c config/config.yaml
+      command:
+        - /bin/sh
+        - -c
+      env:
+        - name: KUBECONFIG
+      envFrom:
+        - secretRef:
+            name: $(params.awsCredentialSecret)
+      resources: {}
+      securityContext:
+        privileged: true
```

Comment:
can we make this Task run without needing privileged access?

Reply:
source example require it https://github.com/chaos-kubox/krkn/blob/main/containers/kraken.yml
I've tested without and it work but not for all chaos scenarios (time, pvc for example).

Reply:
add example for adding appropriate SCC in an Openshift context https://github.com/tektoncd/catalog/pull/1003/commits/c718624f3c52ffa990b701f0e7ca130881f016ce